test/checker.js

```diff
@@ -47,6 +47,14 @@ describe('modules/checker', function() {
                 assert(errors.length === 1);
             });
         });
+
+        it('should throw an exception when path doesn\'t exists', function() {
+            return checker.checkPath('./test/non-exists').then(function() {
```

Comment:
An alternative might be:

```js
return checker.checkPath('./test/non-exists').then(null, done);
```

This requires `done` as a parameter to the `it`, of course